borg/archive.py

```diff
@@ -261,6 +262,7 @@ def save(self, name=None, timestamp=None):
             'username': getuser(),
             'time': start.isoformat(),
             'time_end': end.isoformat(),
+            'chunker_params': self.chunker_params,
```

Comment:
Is this actually a safe thing to do, or can this break older versions trying to read this?